layouts/partials/helpers/get-author-name.html

```diff
@@ -1,7 +1,12 @@
+{{/*  Uses the top level site's config for a single author across all locales  */}}
 {{ $authorName:= site.Params.author.name }}
-{{ if site.Data.site.author}}
-    {{ $authorName = site.Data.site.author.name }}
-{{ end}}
```

Comment:
correct me if i'm wrong, but I don't think the site.Data.site is every populated since there are language-specific site.yml files correct? Or do I need this for passivity for an old style of configuration?

Reply:
Yes. You are right. I am wondering how I missed that.